### PR TITLE
Animal tables

### PR DESCRIPTION
**Pull request by citec-47**
### **- Implementation**
- Find all animals whose name ends in "mon".
names that end with mon
- List the name of all animals born between 2016 and 2019.
born between 2016 and 2019
- List the name of all animals that are neutered and have less than 3 escape attempts.
image
- List the date of birth of all animals named either "Agumon" or "Pikachu".
image
- List name and escape attempts of animals that weigh more than 10.5kg
image
- Find all animals that are neutered.
image
- Find all animals not named Gabumon.
image
- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -227,10 +152,10 @@ Example:
 ## 🔭 Future Features <a name="future-features"></a>
 
 > Describe 1 - 3 features you will add to the project.
```
Comment (Ibizugbe) [inline README.md]: I suppose you left this statement here mistakenly😉. Kindly take it away with every other template statement in your readme file as they give it a less professional look😉.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @citec-47 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] The animal table have been appropriately created in the `schema.sql` file👍.
- [x] Queries have been written and added to the `queries.sql` file👍.
- [x] Data have been added to the animal table and the commands used are written in the `queries.sql` file👍.


## Required Changes ♻️

- [ ] Nice job uploading some of the images of the resukts of your queries on your PR description😉. But let's try to add the result of all our queries as this is a requirement for this milestone👍.
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] Let's try to write more descriptive and meaningful commit messages subsequently as this is more professional😉.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Ndonyi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good job updating the Readme file as requested by the previous CR

 ## Required Changes ♻️
- [ ] as the previous CR mentioned please add a print screen of all the queries you wrote to the PR description, right now there are only 3, if you encounter any issues please leave a comment in the PR.
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @citec-47,  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [ ] Followed Gitflow💯
- [ ] Great job on the functionality🔥🔥
- [ ] Professional README⭐
- [ ] Kudos on adding the screenshots for the queries ⭐


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**